bigtable/hello/main.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates how to connect to Cloud Bigtable and run some basic operations.
+
+Prerequisites:
+
+- Create a Cloud Bigtable cluster.
+  https://cloud.google.com/bigtable/docs/creating-cluster
+- Set your Google Application Default Credentials.
+  https://developers.google.com/identity/protocols/application-default-credentials
+- Set the GCLOUD_PROJECT environment variable to your project ID.
+  https://support.google.com/cloud/answer/6158840
+"""
+
+import argparse
+
+from gcloud import bigtable
+from gcloud.bigtable import happybase
+
+
+def main(project, cluster_id, zone, table_name):
+    column_family_name = 'cf1'
+    column_name = '{fam}:greeting'.format(fam=column_family_name)
+    greetings = [
```

Comment:
Variables moved.

Note: greetings is used only once, but I kept it defined since I think it'd look weird to loop over a naked list. Though with only three items I wonder if just unrolling the loop would be clearer?